src/main/java/emu/grasscutter/server/packet/recv/HandlerGetPlayerTokenReq.java

```diff
@@ -111,6 +111,7 @@ public void handle(GameSession session, byte[] header, byte[] payload) throws Ex
 
         // Only >= 2.7.50 has this
         if (req.getKeyId() > 0) {
+            var encrypt_seed = session.getEncryptSeed();
```

Comment:
rename to encryptSeed?

Reply:
maybe we should rename all the names of variables